dockerCompose/dockerComposeCompletionItemProvider.ts

```diff
@@ -55,12 +59,14 @@ export class DockerComposeCompletionItemProvider implements CompletionItemProvid
         return Promise.resolve([]);
     }
 
-    private suggestKeys(word: string): CompletionItem[] {
-        return Object.keys(DOCKER_COMPOSE_KEY_INFO).map(ruleName => {
+    private suggestKeys(word: string, useV2Schema: boolean = false): CompletionItem[] {
```

Comment:
Can I suggest passing in a version (string or enum) instead of the ```useV2Schema``` boolean?  Then you could do a switch below, vs a ternary to get the appropriate keyset.  This will make this more amenable to future versions of the docker-compose.yml schema.